test/common/prof.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+function getHeapProfiles(dir) {
```

Comment:
Nit: some basic doc comments for each of these utilities added into `test/common` would be helpful. Not blocking tho.